# Enable the next section to build the official 5prealpha release instead of the git version
#--- ./include/pocketsphinx.h.orig	2012-12-12 14:42:03.000000000 -0700
#+++ ./include/pocketsphinx.h	2013-03-29 10:34:44.464096036 -0600
#@@ -339,7 +339,7 @@ int ps_decode_raw(ps_decoder_t *ps, FILE
#  * Decode a senone score dump file.
#  *
#  * @param ps Decoder
#- * @param fh Previously opened file handle positioned at start of file.
#+ * @param senfh Previously opened file handle positioned at start of file.
#  * @return Number of frames read.
#  */
# POCKETSPHINX_EXPORT
--- ./src/libpocketsphinx/dict.h.orig	2012-09-28 16:17:02.000000000 -0600
+++ ./src/libpocketsphinx/dict.h	2013-03-29 10:41:00.320504237 -0600
@@ -81,8 +81,8 @@ typedef struct {
     bin_mdef_t *mdef;	/**< Model definition used for phone IDs; NULL if none used */
     dictword_t *word;	/**< Array of entries in dictionary */
     hash_table_t *ht;	/**< Hash table for mapping word strings to word ids */
-    int32 max_words;	/**< #Entries allocated in dict, including empty slots */
-    int32 n_word;	/**< #Occupied entries in dict; ie, excluding empty slots */
+    int32 max_words;	/**< \#Entries allocated in dict, including empty slots */
+    int32 n_word;	/**< \#Occupied entries in dict; ie, excluding empty slots */
     int32 filler_start;	/**< First filler word id (read from filler dict) */
     int32 filler_end;	/**< Last filler word id (read from filler dict) */
     s3wid_t startwid;	/**< FOR INTERNAL-USE ONLY */
@@ -156,7 +156,7 @@ const char *dict_ciphone_str(dict_t *d,
 /**
  * Number of "real words" in the dictionary.
  *
- * This is the number of words that are not fillers, <s>, or </s>.
+ * This is the number of words that are not fillers, \<s\>, or \</s\>.
  */
 #define dict_num_real_words(d)                                          \
     (dict_size(d) - (dict_filler_end(d) - dict_filler_start(d)) - 2)
--- ./src/libpocketsphinx/acmod.h.orig	2012-09-28 16:17:02.000000000 -0600
+++ ./src/libpocketsphinx/acmod.h	2013-03-29 10:35:10.792056036 -0600
@@ -229,7 +229,7 @@ ps_mllr_t *acmod_update_mllr(acmod_t *ac
  * Start logging senone scores to a filehandle.
  *
  * @param acmod Acoustic model object.
- * @param logfh Filehandle to log to.
+ * @param senfh Filehandle to log to.
  * @return 0 for success, <0 on error.
  */
 int acmod_set_senfh(acmod_t *acmod, FILE *senfh);
